### PR TITLE
docs(general): Switch references to Bridgecrew with Prisma Cloud

### PR DESCRIPTION
**Pull request by tsmithv11**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    Be aware that we use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Additionally a scope is needs to be added to the prefix, which indicates the targeted framework, in doubt choose 'general'.
    #    
    Allowed prefixs:
    ansible|argo|arm|azure|bicep|bitbucket|circleci|cloudformation|dockerfile|github|gha|gitlab|helm|kubernetes|kustomize|openapi|sast|sca|secrets|serverless|terraform|general|graph|terraform_plan|terraform_json
    #
    ex.
    feat(terraform): add CKV_AWS_123 to ensure that VPC Endpoint Service is configured for Manual Acceptance
"

## Description

Switch references to Bridgecrew with Prisma Cloud - purely docs for now

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/common/util/banner.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
  | (__| | | |  __/ (__|   < (_) \ V / 
   \___|_| |_|\___|\___|_|\_\___/ \_/  
                                       
-By bridgecrew.io | version: {} """.format(version)
+By Prisma Cloud | version: {} """.format(version)
```
Comment (tsmithv11) [inline checkov/common/util/banner.py:15]: Will this break anything?

File: tests/common/runner_registry/test_runner_registry.py [hunk 2]
```diff
@@ -737,7 +737,7 @@ def test_output_file_path_with_output_mapping(tmp_path: Path, capsys: CaptureFix
     runner_registry.print_reports(scan_reports=reports, config=config)
 
     # then
-    assert 'By bridgecrew.io' in capsys.readouterr().out
+    assert 'By Prisma Cloud' in capsys.readouterr().out
```
Comment (tsmithv11) [inline tests/common/runner_registry/test_runner_registry.py:740]: And this?

File: checkov/common/util/banner.py (hunk 1 above)
Comment (gruebel) [reply checkov/common/util/banner.py:15]: nope, just the test you had to fix 😄

File: docs/2.Basics/CLI Command Reference.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ nav_order: 2
...
...                                                                                                                                                                                                                                                                                  | When running with an API key, Checkov will omit any policies that do not exist in Prisma Cloud platform, except for local custom policies loaded with the --external-check flags. Use this key to include policies that only exist in Checkov in the scan. Note that this will make the local CLI results different from the results you see in the platform. Has no effect if you are not using an API key. Use the --check option to explicitly include checks by ID even if they are not in the platform, without using this flag.                                                                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (gruebel) [inline docs/2.Basics/CLI Command Reference.md:24]: don't forget to also adjust all the CLI flag help text in `checkov/common/util/ext_argument_parser.py`

Comment (tsmithv11) [reply docs/2.Basics/CLI Command Reference.md:24]: Good callout, thanks! Updated

**Review by gruebel — APPROVED** [review]
🐶

**Review by SteveVaknin — APPROVED** [review]

**Review by AdamDev — APPROVED** [review]

**Pull request merged**